src/components/WrappedActionMenu/WrappedActionMenu.tsx

```diff
@@ -14,6 +14,8 @@ export interface IProps {
   onDeposit: (amount: BN, decimals: number) => () => void
   onWithdraw: (amount: BN, decimals: number) => () => void
   onBurn: (amount: BN, decimals: number) => () => void
+  onWarning: (amount: BN, decimals: number) => () => void
```

Comment:
I still think that these new props are not necessary: just use onWarning callback as onBurn and onBurn as confirmation callback in modal

Reply:
done